.github/workflows/pull-request-snapshot-release.yml

```diff
@@ -0,0 +1,176 @@
+# This workflow will release project with Gradle
+
+name: Release new version for specific pull request (snapshot)
+
+on:
+    workflow_dispatch:
+        inputs:
+            pull_request:
+                description: 'The pull request snapshot that is going to be released (i.e PR-XXXX)'
+                required: true
+
+env:
+    IMAGE_BASE_DIR: ./containers
+
+jobs:
+    build:
+
+        runs-on: ubuntu-latest
+        timeout-minutes: 20
+
+        steps:
+            - uses: actions/checkout@v2
+              with:
+                  ref: ${{ github.head_ref }}
+            - name: Set up JDK 1.8
+              uses: actions/setup-java@v1
+              with:
+                  java-version: 1.8
+            - name: Set up Node
+              uses: actions/setup-node@v2
+              with:
+                  node-version: '14'
+            - run: npm install -g npm@8.1.0
+            - name: Grant execute permission for gradlew
+              run: chmod +x gradlew
+            - name: Cache Gradle packages
+              uses: actions/cache@v2
+              with:
+                  path: |
+                      ~/.gradle/caches
+                      ~/.gradle/wrapper
+                  key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+
+            - name: Cache Node.js modules
+              uses: actions/cache@v2
+              with:
+                  path: |
+                      ~/.npm
+                      **/node_modules
+                  key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json', '**/package.json') }}
+
+            - name: Release with Gradle
+              run: |
+                  BRANCH_NAME=${{ github.event.inputs.pull_request }}
+                  sed -i '/version=/ s/-SNAPSHOT/-'"$BRANCH_NAME"'-SNAPSHOT/' ./gradle.properties
+                  ./gradlew build publishAllVersions -Pzowe.deploy.username=$ARTIFACTORY_USERNAME -Pzowe.deploy.password=$ARTIFACTORY_PASSWORD -Partifactory_user=$ARTIFACTORY_USERNAME -Partifactory_password=$ARTIFACTORY_USERNAME -PpullRequest=$BRANCH_NAME
+              env:
+                  ARTIFACTORY_USERNAME: ${{ secrets.ARTIFACTORY_USERNAME }}
+                  ARTIFACTORY_PASSWORD: ${{ secrets.ARTIFACTORY_PASSWORD }}
+            - name: Cleanup Gradle Cache
+                # Remove some files from the Gradle cache, so they aren't cached by GitHub Actions.
+                # Restoring these files from a GitHub Actions cache might cause problems for future builds.
+              run: |
+                  rm -f ~/.gradle/caches/modules-2/modules-2.lock
+                  rm -f ~/.gradle/caches/modules-2/gc.properties
+
+    build-containers:
```

Comment:
Containers are being published with the wrong version IMAGE_VERSION=1.26.18-SNAPSHOT and to the wrong repository https://zowe.jfrog.io/artifactory/docker-release/ompzowe/